app/Exceptions/Solutions/ManifestDoesntExistSolution.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Pterodactyl\Exceptions\Solutions;
+
+use Spatie\Ignition\Contracts\Solution;
+
+class ManifestDoesntExistSolution implements Solution
+{
+    public function getSolutionTitle(): string
+    {
+        return "The manifest.json file hasn't been generated yet";
```

Comment:
Come on man, can't even use PHP CS Fixer smh.  String should be using single quotes.

Reply:
Didn't want to escape the apostrophe. But also double quotes are fine. It's a common myth that the string interpolation has any effect. Only after running strings in loops for millions of times do you see some millisecond differences between '' and "" strings.

Reply:
I completely missed the apostrophe there, my bad.  We should probably enforce a strict code style at some point, right now we have it setup for dev (and in CI) but it doesn't actually enforce it.